internal/restic/lock.go

```diff
@@ -273,6 +273,7 @@ func RemoveStaleLocks(ctx context.Context, repo Repository) error {
 		}
 
 		if lock.Stale() {
+			(*processed)++
```

Comment:
Shouldn't `processed` only be incremented if the `Remove` operation was successful?